scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/SynchronousSchedulerAgent.java

```diff
@@ -141,10 +141,20 @@ public void schedule(final ScheduleRequest request, final ResultsProtocol.Writer
       writer.succeedWith(results);
     } catch (final SpecificationLoadException e) {
       //unwrap failure message from any anticipated exceptions and forward to subscribers
-      writer.failWith(e.getMessage() + SchedulingCompilationError.schedulingErrorJsonP.unparse(e.errors).toString());
-    } catch (final ResultsProtocolFailure | NoSuchSpecificationException | NoSuchPlanException | IOException | MerlinService.MerlinServiceException e) {
-      //unwrap failure message from any anticipated exceptions and forward to subscribers
-      writer.failWith(e.getMessage());
+      // TODO `SpecificationLoadException` should be serialized as a whole or our data model
```

Comment:
Eventually we will be storing this, but I don't think we have need for it in a fully deserialized state except for on the frontend.

Reply:
Thanks, I've removed this `TODO` since it sounds like we're on track to expand our data model for this structured data.